src/dodal/beamlines/i04.py

```diff
@@ -352,6 +353,20 @@ def oav_to_redis_forwarder() -> OAVToRedisForwarder:
         name="oav_to_redis_forwarder",
         redis_host=REDIS_HOST,
         redis_password=REDIS_PASSWORD,
+        db=7,
+    )
+
+
+@device_factory()
+def murko_results() -> MurkoResultsDevice:
+    """Get the i04 OAV to redis forwarder, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i04, it will return the existing object.
+    """
+    return MurkoResultsDevice(
+        "",
```

Comment:
Could: `prefix` is usually for the PV prefix, `murko_results` doesn't connect to PVs so the prefix is meaningless. I think you can remove it here and in the device.